### PR TITLE
[Example 13] Multi-robot system with lifecycle management

### PR DESCRIPTION
**Pull request by bailaC**
This PR will contain the exact changes for Mutli-Robot Lifecycle example of #133 . It will have merge conflicts resolved.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,9 +128,9 @@ The *RRBot* URDF files can be found in the `urdf` folder of `rrbot_description`
    ros2 launch rrbot_description view_robot.launch.py
    ```
    **NOTE**: Getting the following output in terminal is OK: `Warning: Invalid frame ID "odom" passed to canTransform argument target_frame - frame does not exist`.
-   This happens because `joint_state_publisher_gui` node need some time to start.
-   The `joint_state_publisher_gui` provides a GUI to generate  a random configuration for rrbot. It is immediately displayed in `Rviz`.
+             This happens because `joint_state_publisher_gui` node need some time to start.
```
[the remark this answers is not in the document]
Comment (erickisos) [reply README.md:131]: Maybe we can do this note into a quote block.

File: ros2_control_demo_bringup/launch/three_robots.launch.py [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+def generate_launch_description():
+    # Declare arguments
+    declared_arguments = []
+    declared_arguments.append(
+        DeclareLaunchArgument(
+            "slowdown",
+            default_value="50.0",
+            description="Slowdown factor of the RRbot.",
+        )
+    )
```
Comment (erickisos) [inline ros2_control_demo_bringup/launch/three_robots.launch.py:35]: Why are we defining it and then doing the append?
```suggestion
    declared_arguments = [
        DeclareLaunchArgument(
            "slowdown",
            default_value="50.0",
            description="Slowdown factor of the RRbot.",
        )
    ]
```

File: README.md [hunk 3]
```diff
@@ -508,6 +536,289 @@ Accessing Wrench data from 2D FTS:
 ros2 topic echo /fts_broadcaster/wrench
 ```
 
+### Example XX: "Multi-robot system with tools"
```
Comment (AndyZe) [inline README.md:539]: I think the title should mention "Lifecycle" somehow. Suggestion:

```suggestion
### Example XX: "Multi-robot system with lifecycle management: adjust controllers at runtime"
```

**Review by erickisos — APPROVED** [review]

**Comment by christophfroehlich** [comment]
closed in favor of #417

**Pull request closed without merging**